src/go/k8s/tests/e2e/produce-consume/00-assert.yaml

```diff
@@ -29,9 +29,9 @@ spec:
             - redpanda
             - start
             - --check=false
-            - --reserve-memory 1M
             - --advertise-rpc-addr=$(POD_NAME).cluster-sample.$(POD_NAMESPACE).svc.cluster.local.:33145
             - --overprovisioned
+            - --reserve-memory 1M
```

Comment:
same here, i don't think this has any change.

Reply:
I answered in the previous comment